src/Rules/Library/ChromiumComponentsShouldUseWebScanner.cs

```diff
@@ -27,5 +29,11 @@ protected override Condition CreateCondition()
         {
             return Chrome;
         }
+
+        public override bool IncludeInResults(IA11yElement element)
```

Comment:
Taking a big step back, would it make sense to include this check in `ChronmiumComponentsShouldUseWebScanner.Condition` ? I think that would avoid the whole concept of included and excluded rules, leading to simpler code

Reply:
We could definitely do that, but I originally went with this approach because if we just updated the condition then we will continue to scan the chrome elements for all the other rules, which results in a lot of failures that aren't really reliable. One alternative is we could do the normal scan with the updated condition logic you're suggesting and then find some way to exclude all the failures from all the elements that fail the chrome rule after the fact, but that would mean running tests on many elements just to exclude them later, which felt wasteful to me.